/usb_driver:
  ros__parameters:
    cam_id: 2
    frame_id: usb_cam
    image_width: 640
    image_height: 480
    fps: 30